CHANGELOG.next.asciidoc

```diff
@@ -20,6 +20,10 @@ https://github.com/elastic/beats/compare/v7.0.0-beta1...master[Check the HEAD di
 
 *Metricbeat*
 
+- Migrate docker module to ECS. {pull}10927[10927]
+
+*Packetbeat*
```

Comment:
This entry was actually missing here :thinking:

Reply:
hmm what happened with cherry-pick script hmm